tests/fixtures/transfers/mono_contributor/input/feed_infos.txt

```diff
@@ -1,5 +1,5 @@
 feed_info_param,feed_info_value
-ntfs_version,0.3
+ntfs_version,0.13.0
```

Comment:
Like some others, it's input. 
Normally you didn't need to change it. 
Especially since this ntfs may really be in version 0.10.0.

Reply:
Yes, I realized that, but I'm wondering if we're really testing something with this? If we're not, let's put everything to the same value, easier to maintain (I actually changed all the values at once with a `sed`), what do you think?

Reply:
I would have preferred to leave it as before.
Maybe one day we will have an old version ntfs (with breaking change) in input to guarantee backward compatibility.
But I don't have a definite opinion. We'll see what others say.

Reply:
I've rolled back all the `input` fixtures to previous NTFS versions in f716e462.